src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+$secondary-color: #616070;
+$accent-color : #00ACDC;
+
+.card {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  padding: 16px;
+  padding-top: 32px;
+
+
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    color:  #060B35;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__label {
+    color: $secondary-color;
+    font-size: 12px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    padding: 12px 70px;
+
+
+    background: #FFF;
+    background-color: #00ACDC;
+    border-radius: 5px;
+
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    &:hover {
+      border: 1px solid $accent-color;
+      border-radius: 5px;
+
+      color: $accent-color;
```

Comment:
You should define transparent border and add color on hover
<img width="1015" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/4a98c517-e8e1-4940-abed-da90bd01e485">